index.bs

```diff
@@ -372,6 +372,10 @@ so that the application loads the tiny model in the case of CPU-only devices.
 
 A JavaScript ML framework is responsible for loading, interpreting and executing a ML model. During the model execution phase, the framework iterates through the operations of the model and executes each operation on the hardware device, like CPU, GPU or ML accelerator. To avoid the unnecessary data copying across devices, the framework selects the same device to execute the operations. For a compute intensive operation, such as convolution 2D or matrix multiplication, the framework uses WebNN API to execute it with the ML-specific acceleration available on that selected device.
 
+### Integration with real-time video processing ### {#usecase-real-time-video-processing}
+
+The user experience of WebRTC-based video conferencing is enhanced using real-time video processing. For example, background blur implemented using a [[#usecase-segmentation]] model blurs the background in the user's live camera feed. To satisfy the performance requirements of this use case, the WebNN API integrates with primitives from other Web APIs that make up the media pipeline to allow WebNN API-based transformation of real-time video streams.
+
```

Comment:
Should the full-GPU video processing pipeline be a requirement? For other Web APIs, should we mention WebGPU and WebGL? Because it should require the pre/post-processing are also on the GPU.

Reply:
Thanks for the proposal, that was the key question on my mind also. Let's discuss this on our call to capture the whole group's perspective.